src/main/java/com/ithillel/service/appcontext/PropertiesApplicationContext.java

```diff
@@ -0,0 +1,50 @@
+package com.ithillel.service.appcontext;
+
+import com.ithillel.service.storage.Storage;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+public class PropertiesApplicationContext implements ApplicationContext {
+
+    private static final String TEXT_PROCESSOR = "textProcessor";
+    private Map<String, Object> beans = new HashMap<>();
+
+    public PropertiesApplicationContext() {
+        Properties applicationProperties = new Properties();
+        try {
+            applicationProperties.load(getClass().getClassLoader()
+                    .getResourceAsStream("application.properties"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        for (int i = 0; i < 2; i++) {
```

Comment:
The main goal to make code more generic
Th number 2 in cycle( for (int i = 0; i < 2; i++) {) - is to hardcode value..if properties file will be changed, yours code stop to work at all

That's why you can do:
- change it from "for"to "do-while" 
the stop criteria when  applicationProperties.getProperty("beans[" + i + "].type"); return null as example

or as option you can provide the number of beans(as separate field) in yours property file,  and use it in the code with additional check in null